### PR TITLE
Remove columns that aren't helpful

### PR DESCRIPTION
**Pull request by apmasell**
Particularly, the ones that have callbacks

### CODE REVIEW

File: miso-web/src/main/webapp/scripts/pool_ajax.js [hunk 1]
```diff
@@ -509,21 +504,6 @@ Pool.ui = {
...
-          },
-          {
-            'doOnSuccess': function(json) {
-              jQuery('td:eq(5)', nRow).html(json.response);
-            }
-          }
-        );
-      },
       "aaSorting": [
         [(Sample.detailedSample ? 7 : 0) , "desc"] // NB: this must get updated when adding new columns
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/pool_ajax.js]: Is the index of this still accurate? (and all other hard-coded indices?)

File: miso-web/src/main/webapp/scripts/sample_ajax.js [hunk 2]
```diff
@@ -1064,12 +1064,6 @@ Sample.ui = {
           "iSortPriority": 0
         },
         {
-          "sTitle": "QC Result",
-          "mData": "id",
-          "bSortable": false,
-          "iSortPriority": 0
-        },
-        {
           "sTitle": "Location",
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/sample_ajax.js]: Same issue with hard-coded sorting cols.

File: miso-web/src/main/webapp/scripts/library_ajax.js [hunk 3]
```diff
@@ -1079,10 +1079,6 @@ Library.ui = {
           }
         },
         { 
-          "sTitle": "Type",
-          "mData": "libraryTypeAlias"
-        },
-        { 
           "sTitle": "Sample Name",
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/library_ajax.js]: and hard-coded sorting cols here.

File: miso-web/src/main/webapp/scripts/pool_ajax.js (hunk 1 above)
Comment (apmasell) [inline miso-web/src/main/webapp/scripts/pool_ajax.js]: Fixed. I've added the code that I thought I added before that selects a column by priority.

**Comment by callunity** [comment]
:+1:

**Comment by djcooke** [comment]
:+1:

**Pull request merged**